sap_hana/assets/dashboards/overview.json

```diff
@@ -0,0 +1,586 @@
+{
+    "author_info": {
+        "author_name": "Datadog"
+    },
+    "board_title": "SAP HANA Overview",
+    "created": "2019-10-23T22:03:16.131906+00:00",
+    "created_by": {
+        "access_role": "st",
+        "disabled": false,
+        "email": "support@datadoghq.com",
+        "handle": "support@datadoghq.com",
+        "is_admin": false,
+        "name": "Datadog",
+        "role": null,
+        "verified": true
+    },
+    "description": null,
+    "disableCog": false,
+    "disableEditing": false,
+    "id": 872366,
+    "isIntegration": false,
+    "isShared": false,
+    "modified": "2019-12-05T23:33:09.386413+00:00",
+    "new_id": "aiq-tfk-a4n",
+    "originalHeight": 80,
+    "originalWidth": "calc(100% - 32px)",
+    "read_only": false,
+    "template_variables": [],
+    "widgets": [
+        {
+            "add_timeframe": true,
+            "auto_refresh": false,
+            "bgcolor": "gray",
+            "board_id": "aiq-tfk-a4n",
+            "font_size": "18",
+            "height": 5,
+            "html": "Memory",
+            "isShared": false,
+            "refresh_every": 30000,
+            "scaleFactor": 1,
+            "text_align": "center",
+            "tick": true,
+            "tick_edge": "bottom",
+            "tick_pos": "50%",
+            "title": false,
+            "type": "note",
+            "width": 47,
+            "x": 20,
+            "y": 1
+        },
+        {
+            "add_timeframe": true,
+            "board_id": "aiq-tfk-a4n",
+            "height": 12,
+            "isShared": false,
+            "margin": "",
+            "scaleFactor": 1,
+            "sizing": "fit",
+            "title": false,
+            "type": "image",
+            "url": "/static/images/saas_logos/bot/sap_hana.png",
```

Comment:
Should we favor S3 ? => Less load for our backend servers.

E.g. https://github.com/DataDog/integrations-core/blob/18d61fbd29ab0e4058cb71a77bca12525664caf5/druid/assets/dashboards/overview.json#L57

Reply:
We no longer put logos in s3